src/pages/buy/buy.layout.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { PopupContainer } from '@components/popup/container';
+import { Header } from '@components/header';
+import { Caption, Text, Title } from '@components/typography';
+import { Button, Flex, Stack } from '@stacks/ui';
+import { Link } from '@components/link';
+import AddFunds from '@assets/images/add-funds.svg';
+import { SpaceBetween } from '@components/space-between';
+import { openInNewTab } from '@common/utils/open-in-new-tab';
+import { ActiveFiatProviderType } from '@query/hiro-config/hiro-config.query';
+
+const PROVIDERS_INFO = {
```

Comment:
When are we deciding to use all caps/snake case? I did this in my last PR for some local constants but had them changed by @kyranjamie?

Reply:
Understand the confusion here as there's no hard rule here. I guess the way I look at it is to divide them into two camps: global and local.

**Global**: app wide constant values declared in the constants file, ALL_UPPERCASE convention
**Local**: specific to a component, nothing special, regular camelCase convention.